lightning_pose/data/datasets.py

```diff
@@ -96,13 +96,17 @@ def __init__(
                     raise FileNotFoundError("Could not find csv file!")
                 csv_file = options[0]
 
-        csv_data = pd.read_csv(csv_file, header=header_rows)
-        self.keypoint_names = csv_data.columns.levels[0][1:]  # unused inside class
-
-        self.image_names = list(csv_data.iloc[:, 0])
-        self.keypoints = torch.tensor(
-            csv_data.iloc[:, 1:].to_numpy(), dtype=torch.float32
-        )
+        csv_data = pd.read_csv(csv_file, header=header_rows, index_col=0)
```

Comment:
great update.
we do this in other places, including in a `get_keypoint_names()` function in [the predictions file](https://github.com/danbider/lightning-pose/blob/23dc5f22e4b40fa8b71193322f11fca703fd8ec9/lightning_pose/utils/predictions.py#L486). 
better to maybe unify to one functions in the utils.io and call it here and there too.
are you also sure about this way of reading colnames?